### PR TITLE
SW-1077-increase-laserhead-max-working-temperature

### PR DESCRIPTION
**Pull request by ahmed-mrbeam**
* Temperature manager to use the current Max Temp value for laser head during init procedure
* In case the laser head changes, The temperature_manager will read the new Max temp value based on the new Laser head detected
* Added new iobeam internal event in case of laserhead changed

### CODE REVIEW

File: octoprint_mrbeam/iobeam/temperature_manager.py [hunk 1]
```diff
@@ -70,14 +70,16 @@ def _on_mrbeam_plugin_initialized(self, event, payload):
         self._iobeam = self._plugin.iobeam
         self._analytics_handler = self._plugin.analytics_handler
         self._one_button_handler = self._plugin.onebutton_handler
-
+        self._subscribe()
```
Comment (ahmed-mrbeam) [inline octoprint_mrbeam/iobeam/temperature_manager.py:73]: This change is not related to the fix
it is done because we shouldn't start the timer before all inits are done and the subscribe is and important step during init

**Review by JFSolution — APPROVED** [review]
